Sources/Target/Windows/Process.cpp

```diff
@@ -150,14 +157,18 @@ BOOL Process::debugBreakProcess() {
   }
 
   DWORD threadId;
-  if (CreateRemoteThread(_handle, NULL, 0, (LPTHREAD_START_ROUTINE)address,
-                         NULL, 0, &threadId) == NULL) {
+#if defined(ARCH_ARM)
+  // +1 indicates the thread to start in THUMB mode
+  auto remoteAddress = address + 1;
+#else
+  auto remoteAddress = address;
+#endif
+  if (CreateRemoteThread(_handle, NULL, 0,
+                         (LPTHREAD_START_ROUTINE)remoteAddress, NULL, 0,
+                         &threadId) == NULL) {
     return FALSE;
   }
   return TRUE;
-#else
-  return DebugBreakProcess(_handle);
-#endif
 }
 
 ErrorCode Process::interrupt() {
```

Comment:
This function does nothing besides call `Process::debugBreakProcess`. Can you move the contents of `Process::debugBreakProcess` into `Process::interrupt` and get rid of the former?

When you do that, you'll probably have to make each error checking block return `Platform::TranslateError()` instead of `TRUE` or `FALSE`.